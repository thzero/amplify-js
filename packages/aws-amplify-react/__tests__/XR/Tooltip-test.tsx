import * as React from 'react';
import Tooltip from '../../src/XR/Tooltip';

describe('Tooltip', () => {
	test('renders successfully with no props', () => {
		const wrapper = mount(<Tooltip />);
		expect(wrapper).toMatchSnapshot();
	});

	test('renders successfully with text', () => {
		const wrapper = mount(<Tooltip text="Test Tooltip" />);
		expect(wrapper).toMatchSnapshot();
	});
});
