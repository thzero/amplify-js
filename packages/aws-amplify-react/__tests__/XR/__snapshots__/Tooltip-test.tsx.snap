// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip renders successfully with no props 1`] = `
<Tooltip>
  <div
    className="XR__tooltip___UYyhn"
  />
</Tooltip>
`;

exports[`Tooltip renders successfully with text 1`] = `
<Tooltip
  text="Test Tooltip"
>
  <div
    className="XR__tooltip___UYyhn"
    data-text="Test Tooltip"
  />
</Tooltip>
`;
